osquery/carver/carver.cpp

```diff
@@ -132,6 +132,7 @@ Carver::Carver(const std::set<std::string>& paths,
 
   // Store the path to our archive for later exfiltration
   archivePath_ = carveDir_ / fs::path(kCarveNamePrefix + carveGuid_ + ".tar");
+  compressPath_ = carveDir_ / fs::path(kCarveNamePrefix + carveGuid_ + ".zst");
```

Comment:
nit: would you mind making this:
```
compressPath_ = carveDir_ / fs::path(kCarveNamePrefix + carveGuid_ + ".tar.zst");
```
To indicate that it's a tarred compressed file?